apps/src/feedback.js

```diff
@@ -1024,6 +1039,9 @@ FeedbackUtils.prototype.showSimpleDialog = function (Dialog, options) {
   }
 
   dialog.show();
+  if (textBox) {
+    textBox.focus();
```

Comment:
you can throw in a `textBox.select()` if you want to really emulate the functionality of `prompt`